sphinxcontrib/datatemplates/tests/test_loaders.py

```diff
@@ -0,0 +1,31 @@
+from sphinxcontrib.datatemplates import loaders
+
+
+def test_lookup_json():
+    actual = loaders.loader_for_source('source.json')
+    assert actual == loaders.load_json
+
+
+def test_lookup_yaml():
+    actual = loaders.loader_for_source('source.yaml')
+    assert actual == loaders.load_yaml
+
+
+def test_lookup_yml():
+    actual = loaders.loader_for_source('source.yml')
+    assert actual == loaders.load_yaml
+
+
+def test_lookup_xml():
+    actual = loaders.loader_for_source('source.xml')
+    assert actual == loaders.load_xml
+
+
+def test_lookup_csv():
+    actual = loaders.loader_for_source('source.csv')
+    assert actual == loaders.load_csv
+
+
+def test_lookup_dbm():
+    actual = loaders.loader_for_source('source.dbm')
```

Comment:
It seems there is no file extension for DBM databases as they normally consist of multiple files. See https://en.wikipedia.org/wiki/DBM_(computing)#cite_note-2001-ladd-odonell-xhtml-3